src/domain/Prize.js

```diff
@@ -0,0 +1,62 @@
+export const FIRST_PRIZE = 2000000000;
+export const SECOND_PRIZE = 30000000;
+export const THIRD_PRIZE = 1500000;
+export const FOURTH_PRIZE = 50000;
+export const FIFTH_PRIZE = 5000;
+
+export const PRIZE = {
+  FIRST: {
+    rank: "FIRST",
+    matchCount: 6,
+    prize: FIRST_PRIZE,
+    containBonus: false,
+  },
+  SECOND: {
+    rank: "SECOND",
+    matchCount: 5,
+    prize: SECOND_PRIZE,
+    containBonus: true,
+  },
+  THIRD: {
+    rank: "THIRD",
+    matchCount: 5,
+    prize: THIRD_PRIZE,
+    containBonus: false,
+  },
+  FOURTH: {
+    rank: "FOURTH",
+    matchCount: 4,
+    prize: FOURTH_PRIZE,
+    containBonus: false,
+  },
+  FIFTH: {
+    rank: "FIFTH",
+    matchCount: 3,
+    prize: FIFTH_PRIZE,
+    containBonus: false,
+  },
+  NONE: { rank: "NONE", matchCount: 0, prize: 0, containBonus: false },
+};
+
+class Prize {
+  #rank = [PRIZE.FIRST, PRIZE.SECOND, PRIZE.THIRD, PRIZE.FOURTH, PRIZE.FIFTH];
```

Comment:
```suggestion

class Prize {
  static FIRST = {
    rank: "FIRST",
    matchCount: 6,
    prize: FIRST_PRIZE,
    containBonus: false,
  }
  
  static SECOND = {
    rank: "SECOND",
    matchCount: 5,
    prize: SECOND_PRIZE,
    containBonus: true,
  }
  
  static THIRD = {
    rank: "THIRD",
    matchCount: 5,
    prize: THIRD_PRIZE,
    containBonus: false,
  }
  
  static FOURTH = {
    rank: "FOURTH",
    matchCount: 4,
    prize: FOURTH_PRIZE,
    containBonus: false,
  }
  
  static FIFTH = {
    rank: "FIFTH",
    matchCount: 3,
    prize: FIFTH_PRIZE,
    containBonus: false,
  }
  
  static NONE = { rank: "NONE", matchCount: 0, prize: 0, containBonus: false },
  
  #rank = [Prize.FIRST, Prize.SECOND, Prize.THIRD, Prize.FOURTH, Prize.FIFTH];
```

이렇게 static으로 표현해주고, 바깥에서도 Prize class의 상수를 통해서 가져올 수 있답니다 ㅎㅎ

Reply:
다만 Przie의 상수들도 일정한 규칙을 가지고 있기 때문에, 이것도 하나의 class로 만들어서 관리해줄 수 있지 않을까 싶네요.
```js
class Prize {
  static FIRST = new Rank(...);
}
```
이런 느낌?